sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/HadoopFileDataObject.scala

```diff
@@ -187,14 +187,18 @@ private[smartdatalake] trait HadoopFileDataObject extends FileRefDataObject with
    * List partitions on data object's root path
    */
   override def listPartitions(implicit context: ActionPipelineContext): Seq[PartitionValues] = {
+    getPartitionPathStatis
```

Comment:
```suggestion
    getPartitionPathStatuses
```
picky detail, but in english that s a bit different than in german:
https://www.grammarbook.com/blog/singular-vs-plural/plural-of-status/#:~:text=For%20those%20in%20search%20of,with%20an%20%E2%80%9Ces%E2%80%9D).

Reply:
You're right, changed it to getPartitionPathsStatus